plugins/README.md

```diff
@@ -23,15 +23,15 @@ configuration files.
 
 
 ```sh
-GEOMETRY_PROMPT_PLUGINS=(virtualenv docker_machine exec_time git hg)
+GEOMETRY_PROMPT_PLUGINS=(virtualenv docker_machine exec_time git hg 'rustup*')
 ```
 
 *Note: if you're not sure where to put geometry configs, just add them to your `.zshrc`*
```

Comment:
For the same reasons as before, if I missed the pinning section, I would be confused with the `'rustup*'`. Maybe add a link saying "if you're not sure what's with the `*` you can read about it [here]()" ? What's your take?